pyproject.toml

```diff
@@ -19,8 +19,6 @@ keywords = [
 classifiers = [
   'Development Status :: 3 - Alpha',
   'Programming Language :: Python :: 3',
-  'Programming Language :: Python :: 3.8',
-  'Programming Language :: Python :: 3.9',
```

Comment:
These should not be removed, right? As we want to still support 3.8 (as also described in the `pyproject.toml` file).

Reply:
Oh sorry, I messed up
Will fix now